src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine{
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer is working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer has stopped");

```

Comment:
The message in the `stopWork` method is informative and correctly indicates the type of `Machine` that has stopped.